### ISSUE TITLE
test: VolumeSuite.TestVolumeTransmitAPI failing to delete volume

### ISSUE THREAD

**Issue by josephglanville** [flynn/flynn#3045]
Relevant excerpt:
```
test_volume.go:18:
    s.doVolumeTransmitAPI(hosts[0], hosts[0], t)
test_volume.go:63:
    t.Assert(h1.DestroyVolume(vol2.ID), c.IsNil)
... value httphelper.JSONError = httphelper.JSONError{Code:"unknown_error", Message:"Something went wrong", Detail:json.RawMessage(nil), Retry:false} ("unknown_error: Something went wrong")
```

Full log: https://ci.flynn.io/builds/20160704171721-ced95389

**Comment by titanous** [comment]
The error from the host is:

```text
cannot destroy 'flynn-default/e6886cba-cde8-4b69-be56-8a28120ece0e': dataset is busy
```

**Comment by titanous** [comment]
Let's wait for #3030 before looking into this further, it could be related to the mount leaks we've seen with libvirt-lxc.

**Comment by titanous** [comment]
I believe this was fixed by #3030.
